### DISCUSSION TITLE
The Phenomenon of Latency Jumps in Inference

### DISCUSSION THREAD

**Discussion by UltraTar0** [vllm-project/vllm#8202] [Ideas]
When I used vLLM to accelerate inference for Llama3 70b, I observed an interesting phenomenon. When each input request consists of 1 token, there is a latency jump every 256 requests. Does anyone know why this happens?
![llama370bcuda8](https://github.com/user-attachments/assets/961fb1bf-cb13-458e-9c64-138cc92dccfa)

**Comment by SeungminHeo** [comment]
because default scheduler batch size is `256` so when reached 256 requests and nothing completed, additional requests will be Waited by scheduler within vLLM. so waiting time is added to latency, that is the phenomenon you observed i guess.

**Reply by UltraTar0** [reply]
Thank you！
